wordssite/dictionary/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+from .models import Word,Translation
+from django.core.exceptions import ValidationError
+ 
+ 
+class WordForm(forms.ModelForm):
+    def clean_word(self):
+        word = self.cleaned_data["word"].title()
+
+        if Word.objects.filter(word = word).count()>0:
```

Comment:
`if Word.objects.filter(word = word).exists():` would be more optimal here. check this out:
https://dev.to/codereviewdoctor/why-queryset-exists-is-more-efficient-than-queryset-count-2f3h#:~:text=The%20Django%20docs%20tells%20us,this%20advice%20is%20because%20queryset.